data/json/mutations/mutations.json

```diff
@@ -466,7 +467,6 @@
     "points": -1,
     "description": "You tend to have bouts of depression during the fall and especially winter.  Taking antidepressants can help.",
     "starting_trait": true,
-    "types": [ "ATTITUDE" ],
```

Comment:
Why editing the type of another unrelated trait here?

Reply:
fairyarmadillo commented May 8, 2024 Giving this ATTITUDE means that Plant will eternally flipflop between gaining and losing seasonal affective disorder 
bateau-de-coal commented May 8, 2024 Seasonal affective disorder: a solution could be making seasonal affective disorder not an ATTITUDE, as it is not a psychological problem.
Kamejeir commented May 8, 2024 Seasonal Affective Disorder can have its category removed, but don't forget to add it to the list of stuff Human Mind cancels or tests will give you a hard time.